src/napari_workflows/_workflow.py

```diff
@@ -219,8 +222,22 @@ def update_layer(whatever):
                         self.viewer.layers[name].data = data
 
             # Start the loop
-            worker.yielded.connect(update_layer)
-            worker.start()
+            self.worker.yielded.connect(update_layer)
+            self.worker.start()
+
+    def pause_update(self) -> None:
+        """Temporarily halt the automatic update."""
+        self.worker.pause()
+        self._is_active = False
+
+    def resume_update(self) -> None:
+        """Resume the automatic update."""
+        self.worker.resume()
+        self._is_active = True
+
+    def is_paused(self) -> bool:
+        """Return the current worker state."""
+        return self._is_active
```

Comment:
```suggestion
    def is_autoupdate_paused(self) -> bool:
        """Return the current worker state."""
        return not self._is_active
```

It took me a few moments to realized that `is_update_paused` is not an or question ("Is updated or paused?) rather than "is the updating paused?" Would you mind if I changed the function's name like this?

Reply:
@haesleinhuepf just pinging you here - are you ok with this function naming? If this is merged I could go ahead and add a play/pause button in the assistant and make the function available :)